amf-cli/shared/src/test/resources/semantic-jsonschema/dialects/allOf-with-extended-schema.yaml

```diff
@@ -0,0 +1,39 @@
+#%Dialect 1.0
+nodeMappings:
+  Child:
```

Comment:
Weren't we going to add something to avoid creating this empty mapping ? Don't really care if it is left empty, just asking. Asking because the root schema has properties

Reply:
I think is better like this, to have the same behavior in all cases.